app/app.rb

```diff
@@ -282,7 +282,7 @@ def render_directory(directory)
 		def process_form_data(data, git)
 			case data
 			when Array
-				data.map { |v| process_form_data(v, git) }
+				data.reject { |v| v.is_a?(String) && v.blank? }.map { |v| process_form_data(v, git) }
```

Comment:
It is ActiveSupport. I do like these convenience methods, but I understand not wanting to bring the monkey patches in. Do you have a suggestion for how we might get some of the convenience without that?